src/resources/dispatch/ReportData.tsx

```diff
@@ -10,11 +10,33 @@ import FlexBox from '../../components/FlexBox'
 
 const DispatchDetail = (): React.ReactElement => {
   return (
-    <Show component={'div'} actions={<></>}>
-      <FieldWithLabel label='Serial No.' source='reference' />
-      <FieldWithLabel label='From' source='createdAt' />
-      <FieldWithLabel label='Date' source='createdAt' />
-      <FieldWithLabel label='To' source='toName' />
+    <Show component={'div'} actions={<></>} sx={{ marginBottom: '10px' }}>
+      <FieldWithLabel
+        label='Serial No.'
+        source='reference'
+        labelStyles={{ fontSize: '1rem' }}
+        textProps={{ variant: 'h6', sx: { fontSize: '1rem', width: '50%' } }}
+      />
+      <FieldWithLabel
```

Comment:
No, @TahaKhanAbdalli  - the from address is a fixed address for the whole app.

See `Dept A, Building A... ` in the screenshot in the parent issue.